src/fsharp/Logger.fsi

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Microsoft.FSharp.Compiler
+
+// FIXME: We cannot make this internal yet until F# gets a compiler switch to make cases public when the type itself is internal.
+// https://github.com/Microsoft/visualfsharp/issues/4821
+type (* internal *) LogCompilerFunctionId =
+    | Parse_And_Check_File_In_Project = 1
+
+[<RequireQualifiedAccess>]
+module internal Logger =
+#if ETW_LOGGING
+    val Log : LogCompilerFunctionId -> unit
+
+    val BlockStart : LogCompilerFunctionId -> unit
```

Comment:
Any reason not to `inline` these cases too?

Reply:
No, we don't need to inline these. I wanted the others inlined as they wouldn't output any IL.